microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -466,6 +466,8 @@ If you previously saved payment information to your Microsoft account, it is als
 
 The feature is only available for users with a non-child Microsoft account.  The feature is not available for users signed into Microsoft Edge with a work or school account.
 
+<!-- ====================================================================== -->
+## Personalization
```

Comment:
```suggestion

<!-- ====================================================================== -->
## Personalization

```
per repo conventions for readability/ease of maintenance and for well-formed robust Markdown.